src/js/helpers/treemapHelper.js

```diff
@@ -54,13 +54,32 @@ export const tooltipStyles = {
 };
 
 export const stateTreemapColors = [
-    "#053547",
-    "#325566",
-    "#587786",
-    "#7E9BA8",
-    "#A6C2CB"
+    "#053547", // direct payments
+    "#325566", // grants
+    "#587786", // contracts
+    "#7E9BA8", // other financial assistance
+    "#A6C2CB" // loans
```

Comment:
the color codes map to the amount from largest to smallest not the award type so we should remove these comments